redbot/setup.py

```diff
@@ -272,12 +272,14 @@ def basic_setup():
                 raise NotImplementedError("We cannot convert from JSON to MongoDB at this time.")
                 # if confirm("Would you like to import your data? (y/n) "):
                 #    await json_to_mongo(current_data_dir, storage_details)
-        else:
+        elif storage_dict.get(storage, 1) == "JSON":
             storage_details = instance_data["STORAGE_DETAILS"]
             default_dirs["STORAGE_DETAILS"] = {}
             if instance_data["STORAGE_TYPE"] == "MongoDB":
                 if confirm("Would you like to import your data? (y/n) "):
                     await mongo_to_json(current_data_dir, storage_details)
+            elif instance_data["STORAGE_TYPE"] == "MongoDBV2":
+                raise NotImplementedError("We cannot convert from MongoDB2 to JSON at this time.")
```

Comment:
```suggestion
                raise NotImplementedError("We cannot convert from MongoDB to JSON at this time.")
```
I don't think putting a 2 next to this is clear to the user. It has meaning internally to us, but shouldn't be user-facing.

Beyond that, I think this is ready to go, so since this is just cosmetic in nature, take or leave the change, it's fine.

Reply:
The only reason I kept that in there is because Mongo and MongoV2 are separate options in the conversion menu and I think they need to be. I'll take another look at it and see if I can condense them.